### PR TITLE
update phpstan to 0.11

### PR DESCRIPTION
**Pull request by adaamz**
There is issue which I dont know how you want to resolve:
`Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php:`
```
Array has 2 duplicate keys with value '=' (\Doctrine\Common\Collections\Expr\Comparison::EQ, \Doctrine\Common\Collections\Expr\Comparison::IS).
```

Just tell me which one I should delete and I fix it in this PR.

### CODE REVIEW

**Comment by carusogabriel** [comment]
@CZechBoY I believe that one should be ignored. Whoever is using `Comparison::EQ` or `Comparison::IS` is gonna be a BC 😕

**Comment by CZechBoY** [comment]
@carusogabriel The collision is in private static property, but keys and values are the same.
```
class BasicEntityPersister implements EntityPersister
{
    /** @var string[] */
    private static $comparisonMap = [
        Comparison::EQ          => '= %s',
        Comparison::IS          => '= %s',
        Comparison::NEQ         => '!= %s',
        Comparison::GT          => '> %s',
        Comparison::GTE         => '>= %s',
        Comparison::LT          => '< %s',
        Comparison::LTE         => '<= %s',
        Comparison::IN          => 'IN (%s)',
        Comparison::NIN         => 'NOT IN (%s)',
        Comparison::CONTAINS    => 'LIKE %s',
        Comparison::STARTS_WITH => 'LIKE %s',
        Comparison::ENDS_WITH   => 'LIKE %s',
    ];
```

So should I remove IS/EQ? Or change constant value?

File: lib/Doctrine/ORM/Tools/Pagination/CountOutputWalker.php [hunk 1]
```diff
@@ -49,7 +50,7 @@ class CountOutputWalker extends SqlWalker
      * @param ParserResult $parserResult
```
Comment (carusogabriel) [inline lib/Doctrine/ORM/Tools/Pagination/CountOutputWalker.php]: You can drop these two phpDocs.

File: lib/Doctrine/ORM/Query/TreeWalkerChain.php [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 use function array_diff;
 use function array_keys;
+use Doctrine\ORM\AbstractQuery;
```
Comment (carusogabriel) [inline lib/Doctrine/ORM/Query/TreeWalkerChain.php]: This should go before the `use function *` section.

File: lib/Doctrine/ORM/Tools/Pagination/LimitSubqueryOutputWalker.php [hunk 3]
```diff
@@ -89,7 +90,7 @@ class LimitSubqueryOutputWalker extends SqlWalker
      * @param ParserResult $parserResult
```
Comment (carusogabriel) [inline lib/Doctrine/ORM/Tools/Pagination/LimitSubqueryOutputWalker.php]: You can remove these two phpDocs.

**Review by carusogabriel — CHANGES_REQUESTED** [review]
> So should I remove IS/EQ? Or change constant value?

Let's add it to the `phpstan.neon` file under `ignoreErrors` for now.

**Pull request closed without merging**